README.md

```diff
@@ -240,6 +240,19 @@ db_password:
   secret: db_password
 ```
 
+### Parameter Store
+An alternative to the secrets store, uses the AWS SSM Parameter store to protect
+secrets.   Expects a parameter of either `String` or `SecureString` type to be present in the
+same region as the stack. You can store the parameter using a command like this
+`aws-vault exec <whatever> -- aws ssm put-parameter --region <region> --name <parameter name> --value <secret> --type (String|SecureString)`
```

Comment:
I don't think we should reference `aws-vault` here, as many people use different authentication methods.

Reply:
This needs some more white space to avoid being included on the same line